tests/testArgumentParser.py

```diff
@@ -289,13 +290,15 @@ def testAtFile(self):
     def testLogDest(self):
         """Test --logdest
         """
-        logFile = "tests_argumentParser_testLog_temp.txt"
-        self.ap.parse_args(
-            config = self.config,
-            args = [DataPath, "--logdest", logFile],
-        )
-        self.assertTrue(os.path.isfile(logFile))
-        os.remove(logFile)
```

Comment:
I can confirm that `psutil.Process().open_files()` does not return the log file but `/usr/sbin/lsof` does list the log file as being open still.